action.yaml

```
@@ -9,17 +9,28 @@ inputs:
   skip_push:
     required: false
     default: false
+  read_checksum_token:
+    required: false
+    default: ''  # empty string replaced by `env.GITHUB_TOKEN`
+    description: |
+      This token to be used when executes `aqua update-checksum`. It must have `contents:read` permission about all repositories
+      in tools managed by `aqua`. By default, `env.GITHUB_TOKEN` is used.
+      This input is useful to fetch checksum from private repositories.
 runs:
   using: composite
   steps:
     - shell: bash
       working-directory: ${{inputs.working_directory}}
       run: aqua update-checksum -deep
       if: inputs.prune != 'true'
+      env:
+        GITHUB_TOKEN: ${{ (inputs.read_checksum_token != '') &&  inputs.read_checksum_token || env.GITHUB_TOKEN }}
```

```suggestion
        AQUA_GITHUB_TOKEN: ${{ (inputs.read_checksum_token != '') &&  inputs.read_checksum_token || env.AQUA_GITHUB_TOKEN }}
```

`AQUA_GITHUB_TOKEN` takes precedence over `GITHUB_TOKEN`.
And if it isn't set or empty, `GITHUB_TOKEN` is used.

https://github.com/aquaproj/aqua/blob/b38d21f6967316d32277b20281eebc3e60366be7/pkg/github/github.go#L40-L45

---

Fixed! [`5b0c97e` (#79)](https://github.com/aquaproj/update-checksum-action/pull/79/commits/5b0c97e303c34df228c5eec36342f1f7d123989d)